### PR TITLE
stewardscript: insert Special:GlobalUserRights link on CentralAuth

### LINKED ISSUE #129: [stewardscript] Add 'Special:GlobalUserRights' link on CentralAuth
Convenient for steward work. PR incoming (if I remember how this is done on GitHub).

### PR DESCRIPTION
**Pull request by MarcoAurelioWM**
Fixes #129

### CODE REVIEW

File: user-scripts/pathoschild.stewardscript.js [hunk 1]
```diff
@@ -145,6 +145,12 @@
                                 .text("crossactivity")
                                 .attr({ href: "//tools.wmflabs.org/meta/crossactivity/" + encodeURIComponent(user), title: "Pathoschild's CrossActivity (measures a user's latest edit, bureaucrat, or sysop activity on all wikis)" })
                             )
+                            .append(", ")
+                            .append(this
+                                .Make("a")
+                                .text("global user rights")
+                                .attr({ href: "//meta.wikimedia.org/wiki/Special:GlobalUserRights" + encodeURIComponent(user), title: "Global User Rights" })
```
Comment (Pathoschild) [inline user-scripts/pathoschild.stewardscript.js]: There's a missing slash between `Special:GlobalUserRights` and the username here.

Comment (MarcoAurelioWM) [reply user-scripts/pathoschild.stewardscript.js]: Oh, that's right. I'll amend the patch. Thanks for reviewing :)

Comment (MarcoAurelioWM) [reply user-scripts/pathoschild.stewardscript.js]: Fixed on 01cbf01 I think.

**Comment by MarcoAurelioWM** [comment]
I think I fixed the missing `/` issue but I'm not sure I did it right. I'm not used to GitHub PRs 😸

**Comment by Pathoschild** [comment]
Thanks for the pull request! Changes merged and deployed.

**Comment by MarcoAurelioWM** [comment]
Thank you. Checking `Special:CentralAuth` now but I don't see the link over there. Did I miss some other code, or is `tools-static` just lagging behind?

**Comment by Pathoschild** [comment]
(I'll reply on #129, since the PR itself is merged.)

**Pull request merged**